src/main/java/cn/nukkit/command/defaults/TransferServerComand.java

```diff
@@ -3,11 +3,12 @@
 import cn.nukkit.command.CommandSender;
 import cn.nukkit.command.ConsoleCommandSender;
 import cn.nukkit.command.data.CommandParameter;
+import cn.nukkit.event.player.PlayerTransferEvent;
 import cn.nukkit.lang.TranslationContainer;
 import cn.nukkit.Player;
 import cn.nukkit.utils.TextFormat;
 
-public class TransferServerCommand extends VanillaCommand {
+class TransferServerCommand extends VanillaCommand {
```

Comment:
[ERROR] /home/travis/build/Nukkit/Nukkit/src/main/java/cn/nukkit/command/defaults/TransferServerComand.java:[10,8] class TransferServerCommand is public, should be declared in a file named TransferServerCommand.java


Why?